src/app/static/src/app/components/header/UserHeader.vue

```diff
@@ -77,7 +77,7 @@
                     return filename;
                 }),
             isGuest() {
-                return this.user == "guest"
+                return this.$store.getters.isGuest
```

Comment:
This absolutely works, but isn't consistent with the codebase. We use the `mapGetters` helper (much like the `mapState`, `mapActions`, `mapMutations` helpers.) 

See https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue#L161 for an example. In this case, because it is a root getter the code will be even simpler than that example:

```
isGuest() {
     return this.$store.getters.isGuest         
}
```
becomes
```
...mapGetters(["isGuest"])
```

Reply:
I've applied this to every instance of isGuest except in the ADRIntegration.vue where TS seems to have issues with a computed property loggedIn returning a getter